cmd/system-probe/config/config_unsupported.go

```diff
@@ -18,3 +18,7 @@ const (
 func ValidateSocketAddress(sockPath string) error { //nolint:revive // TODO fix revive unused-parameter
 	return fmt.Errorf("system-probe unsupported")
 }
+
+func canEnablePES() bool {
+	return true
```

Comment:
should this be `false` since PES requires linux?

Reply:
also please add a comment to either/both functions with what the acronym `PES` is

Reply:
Alternatively, use a longer function name like: `processEventDataStreamSupported() bool`